Use Boost.Process with mingw32

--- a/boost/process/detail/windows/basic_cmd.hpp
+++ b/boost/process/detail/windows/basic_cmd.hpp
@@ -159,8 +159,8 @@ struct exe_cmd_init : handler_base_ext
         return exe_cmd_init<Char>(std::move(sh), std::move(args_));
     }
 
-    static std:: string get_shell(char)    {return shell(). string(codecvt()); }
-    static std::wstring get_shell(wchar_t) {return shell().wstring(codecvt());}
+    static std:: string get_shell(char)    {return shell(). string(); }
+    static std::wstring get_shell(wchar_t) {return shell().wstring();}
 
     static exe_cmd_init<Char> cmd_shell(string_type&& cmd)
     {
